import/translation_maps/format_map.properties

```diff
@@ -12,6 +12,7 @@ Chart               = Photo
 ChipCartridge       = Software
 Collage             = Photo
 ConferenceProceeding= Conference Proceeding
+Dataset             = Dataset
```

Comment:
I think "Data Set" is more common usage than "Dataset," though maybe that's an American perspective.

Reply:
"Data Set" is good enough for Wikipedia https://en.wikipedia.org/wiki/Data_set